### PR TITLE
perf: slightly optimize config/config.go & fix: channel`s closing parser/parser.go

### PR DESCRIPTION
**Pull request by CoderI421**
perf: slightly optimize config/config.go
- Exclude the struct tag is space
- Exclude configuration file comment, which is starting with space

fix: in the case of panic unexpected happened in the method parse0 of parser/parser.go, closing of the channel is needed to prevent the program from getting stuck~

Limited ability, hope to make it better~

### CODE REVIEW

File: redis/parser/parser.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func parse0(reader io.Reader, ch chan<- *Payload) {
 	defer func() {
 		if err := recover(); err != nil {
 			logger.Error(err, string(debug.Stack()))
+			ch <- nil
```
Comment (HDT3213) [inline redis/parser/parser.go:79]: close(ch) may be better

**Comment by HDT3213** [comment]
Could you squash 4 commits into 1 commit? 

Besides, We have merged commit 'slightly optimize the channel type to save memory space in file wait.go', haven't we? why does this commit appear again?

File: redis/parser/parser.go (hunk 1 above)
Comment (CoderI421) [reply redis/parser/parser.go:79]: yes, agree with you~ I don't know if it's useful for anything else, so I used "nil", i will fix it and re-PR

**Comment by CoderI421** [comment]
> Could you squash 4 commits into 1 commit?
> 
> Besides, We have merged commit 'slightly optimize the channel type to save memory space in file wait.go', haven't we? why does this commit appear again?

I will try~

**Pull request closed without merging**